driver/registry_memory.go

```diff
@@ -138,9 +143,23 @@ func (r *RegistryMemory) CredentialHandler() *api.CredentialsHandler {
 	return r.ch
 }
 
+func (r *RegistryMemory) HealthEventManager() health.EventManager {
+	if r.healthEventManager == nil {
+		var err error
+		rulesReadinessChecker := rulereadiness.NewReadinessHealthChecker()
+		if r.healthEventManager, err = health.NewDefaultHealthEventManager(rulesReadinessChecker); err != nil {
+			r.logger.WithError(err).Error("unable to instantiate new health event manager")
```

Comment:
I think it is fine to fatal at this point and exit. If there is an error it would always happen on initialization right?
```suggestion
			r.logger.WithError(err).Fatal("unable to instantiate new health event manager")
```

Reply:
Yes exactly